From c2256cd70161db8f8224561391fa2fe77c380ddb Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Wed, 16 Jan 2019 12:12:32 -0800
Subject: [PATCH] Drop using '_' in version

As per debian versioning policy we can only use [a-z0-9.+-]
this creates packaging problems for opkgs as well which follows same
policy for versioning ( in OpenEmbedded )

*** Error: Package name kernel-module-can-isotp-4.14.73-linux4sam_6.0 contains illegal characters, (other than [a-z0-9.+-])

Changing "_" to "-" helps fix this issue and adheres to debian
versioning policy at same time

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index a151b7249ca3..ed824f23c134 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 4
 PATCHLEVEL = 14
 SUBLEVEL = 73
-EXTRAVERSION = -linux4sam_6.0
+EXTRAVERSION = -linux4sam-6.0
 NAME = Petit Gorille
 
 # *DOCUMENTATION*
-- 
2.20.1

